policy-test/src/lib.rs

```diff
@@ -168,6 +168,14 @@ pub async fn await_route_status(
     .inner
 }
 
+// Wait for the endpoints controller to populate the Endpoints resource.
```

Comment:
nit, not a big deal:
```suggestion
/// Returns `true` if the endpoints controller has populated an Endpoints resource.
```

Reply:
we could leave the "// wait for..." comment in the places where this is actually used with `await_condition`, if you want, but not a big deal either way.